python/saw/exceptions.py

```diff
@@ -82,6 +84,8 @@ class CryptolError(SAWException): pass
     10050: NotAnLLVMMethodSpecIR,
     10060: NotASimpset,
     10070: NotATerm,
+    10080: NotAJVMClass,
```

Comment:
These should be added to `docs/Errors.rst` as well. Perhaps there's a good way to have one data structure that generates both the docs and these tables.

Reply:
Ah, good. I'll add those.